storage/diskmetricstore.go

```diff
@@ -123,18 +128,15 @@ func (dms *DiskMetricStore) GetMetricFamilies() []*dto.MetricFamily {
 					result[stat.pos] = existingMF
 				}
 				if mf.GetHelp() != existingMF.GetHelp() {
-					log.Infof(
-						"Metric families '%s' and '%s' have inconsistent help strings. The latter will have priority. This is bad. Fix your pushed metrics!",
-						mf, existingMF,
-					)
+					level.Info(dms.logger).Log("msg", "metric famlies have inconsistent help strings", "new", mf, "old", existingMF)
 				}
 				// Type inconsistency cannot be fixed here. We will detect it during
 				// gathering anyway, so no reason to log anything here.
 				existingMF.Metric = append(existingMF.Metric, mf.Metric...)
 			} else {
 				copied := false
 				if help, ok := dms.predefinedHelp[name]; ok && mf.GetHelp() != help {
-					log.Infof("Metric family '%s' has the same name as a metric family used by the Pushgateway itself but it has a different help string. Changing it to the standard help string %q. This is bad. Fix your pushed metrics!", mf, help)
+					level.Info(dms.logger).Log("msg", "metric family has the same name as a metric family used by the Pushgateway itself but it has a different help string. Changing it to the standard help string. This is bad. Fix your pushed metrics!", "metric", mf, "help", help)
```

Comment:
Hmm, yeah, that's a special case where we have whole sentences. I would probably start with capital “Metric” in this case.

The logging key names should probably be the following to avoid misunderstandings:
- `"metric"` → `"metric_family"`
- `"help"` → `"standard_help"`